### DISCUSSION TITLE
[STUDY GROUP]: GitHub Administration Exam

### DISCUSSION THREAD

**Discussion by v-fidelusaleksander** [v-fidelusaleksander/ghcertified#167] [Study Together!]
# GitHub Administration :book: STUDY GROUP

Hello everyone,

Many of us are preparing for the GitHub Certification Exams and I thought it would be beneficial to start a discussion where we can share:

-  experiances
-  study tips
-  ask questions 
- anything else regarding any of the topics covered in the exam 

Looking forward to hearing from you all! :rocket:

**Comment by christiant-infomagnus** [comment]
Hi,
I've just passed the exam with an 82%.
There are few questions that are exactly as the Practice Test but there are more questions that a bit more "complicated".
Something like (i dont remember exactly):
**If you want to prevent pushing merge commits, which branch protection rule should work?**
or
**Which policies should let users fork internal or private repositories?** (you had to choose 2 or 3)
or
Sometimes you can have questions that needs 3 answers that are the steps for the complete solution.

I thought this wouldn't go deeply in some topics but It went. For example the rules of the branches or knowing that is recommended, at leats, 2 Org Owners.

I will try to remember some of the questions to add them to the practice test.
If you are trying to pass the certification, I recommend doing them.

**Reply by v-fidelusaleksander** [reply]
Hey @christiant-infomagnus 

Congratulations on passing your exam 🎉 

As noted in the [CONTRIBUTING](https://github.com/FidelusAleksander/integrate-mcp-with-copilot) guide of this project.

> [!WARNING]
> We do not support the inclusion of questions directly copied from official GitHub certification exams. Please only submit original questions and content that you have created.
